### PR TITLE
Migrate command `autorecovery`

### PR DESCRIPTION
**Pull request by zymap**
Descriptions of the changes in this PR:

- Using `bkctl` run command `autorecovery`

### Motivation

#2009

### CODE REVIEW

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/toggle/AutoRecoveryCommand.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+     */
+    @Accessors(fluent = true)
+    @Setter
+    public static class AutoRecoveryFlags extends CliFlags {
+
+        @Parameter(names = { "-e", "--enable" }, description = "Enable or disable auto recovery of under replicated "
+                                                               + "ledgers.")
+        private boolean enable;
+
+        @Parameter(names = {"-l", "--look"}, description = "Check the auto recovery status.")
```
Comment (jiazhai) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/toggle/AutoRecoveryCommand.java]: nit: change look to status?

Comment (zymap) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/toggle/AutoRecoveryCommand.java]: yeah, fixed

File: tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/ToggleCommandGroup.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+import org.apache.bookkeeper.tools.cli.commands.toggle.AutoRecoveryCommand;
+import org.apache.bookkeeper.tools.common.BKFlags;
+import org.apache.bookkeeper.tools.framework.CliCommandGroup;
+import org.apache.bookkeeper.tools.framework.CliSpec;
+
+/**
+ * Commands on some specific operation.
+ */
+public class ToggleCommandGroup extends CliCommandGroup<BKFlags> {
```
Comment (sijie) [inline tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/ToggleCommandGroup.java]: I think the command group should be `autorecovery` and the command should be 'toggle', no?

Comment (zymap) [reply tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/ToggleCommandGroup.java]: yeah, thank you! fix later

**Comment by zymap** [comment]
run integration tests

**Comment by sijie** [comment]
@zymap can you rebase to latest master?

**Pull request merged**